dandi/tests/test_validate.py

```diff
@@ -1,35 +1,58 @@
 from glob import glob
+import json
 import os
+import pathlib
 
-import appdirs
+import pytest
 
+from .fixtures import BIDS_TESTDATA_SELECTION
+from ..validate import validate_bids
 
-def test_validate_bids(bids_examples, tmp_path):
-    from ..validate import validate_bids
 
-    # TEST DEFAULT REPORT PATH:
-    # Replace explicit dataset with `os.listdir(bids_examples)[1]` whenever we
-    # implement light data cloning analogous to (`[1]` because `[0]` is .git):
-    # https://github.com/bids-standard/bids-specification/pull/1143
-    selected_dataset = os.path.join(bids_examples, "asl003")
-    _ = validate_bids(selected_dataset, report=True)
+@pytest.mark.parametrize("dataset", BIDS_TESTDATA_SELECTION)
+def test_validate_bids(bids_examples, tmp_path, dataset):
+
+    selected_dataset = os.path.join(bids_examples, dataset)
+    validation_result = validate_bids(selected_dataset, report=True)
+    for i in validation_result:
+        assert not hasattr(i, "severtiy")
```

Comment:
@TheChymera Is this typo deliberate?  If it is, what's the point?  If it's not, shouldn't you instead be testing that the attribute is `None`?

Reply:
@jwodder thank you for spotting this, indeed it was an error, and further, it was giving a false positive arising from an upstream oversight. Will sumbit a fix PR once the [reference data is fixed](https://github.com/bids-standard/bids-examples/pull/338), since currently fixing this would fail due to the data being broken.

Reply:
@jwodder fix is here → https://github.com/dandi/dandi-cli/pull/1148